safety_limiter/src/safety_limiter.cpp

```diff
@@ -602,6 +604,21 @@ class SafetyLimiterNode
     return out;
   }
 
+  geometry_msgs::Twist
+  limitCentrifugalAcceleration(const geometry_msgs::Twist& in)
+  {
+    auto out = in;
+    const double out_linear_vel = std::hypot(out.linear.x, out.linear.y);
```

Comment:
```suggestion
    const double in_linear_vel = std::hypot(in.linear.x, in.linear.y);
```
`out` will be calculated later